src/test/java/io/stargate/sgv2/jsonapi/api/v1/RangeReadIntegrationTest.java

```diff
@@ -0,0 +1,419 @@
+      int[] ids = {1, 2, 3};
+      List<Object> testDatas = getDocuments(ids);
+      String json =
+          """
+        {
+          "find": {
+            "filter" : {"userId" : {"$lte" : 3}},
+            "sort" : {"userId" : 1}
+          }
+        }
+        """;
+      JsonNodeFactory nodefactory = objectMapper.getNodeFactory();
+      final ArrayNode arrayNode = nodefactory.arrayNode(testDatas.size());
+      for (int i = 0; i < testDatas.size(); i++)
+        arrayNode.add(
+            objectMapper.readTree(
+                objectMapper
+                    .writerWithDefaultPrettyPrinter()
+                    .writeValueAsString(testDatas.get(i))));
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(json)
+          .when()
+          .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
+          .then()
+          .statusCode(200)
+          .body("status", is(nullValue()))
+          .body("errors", is(nullValue()))
+          .body("data.documents", hasSize(3))
+          .body("data.documents", jsonEquals(arrayNode.toString()));
+    }
+
+    @Test
+    @Order(7)
+    public void rangeWithDate() throws Exception {
+      int[] ids = {24, 25};
+      List<Object> testDatas = getDocuments(ids);
+      String json =
+          """
+        {
+          "find": {
+            "filter" : {"dateValue" : {"$gt" : {"$date" : 1672531223000}}},
+            "sort" : {"userId" : 1}
+          }
+        }
+        """;
+      JsonNodeFactory nodefactory = objectMapper.getNodeFactory();
+      final ArrayNode arrayNode = nodefactory.arrayNode(testDatas.size());
+      for (int i = 0; i < testDatas.size(); i++)
+        arrayNode.add(
+            objectMapper.readTree(
+                objectMapper
+                    .writerWithDefaultPrettyPrinter()
+                    .writeValueAsString(testDatas.get(i))));
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(json)
+          .when()
+          .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
+          .then()
+          .statusCode(200)
+          .body("status", is(nullValue()))
+          .body("errors", is(nullValue()))
+          .body("data.documents", hasSize(2))
+          .body("data.documents", jsonEquals(arrayNode.toString()));
+    }
+
+    @Test
+    @Order(8)
+    public void rangeWithText() throws Exception {
+      int[] ids = {24, 25};
+      List<Object> testDatas = getDocuments(ids);
+      String json =
+          """
+        {
+          "find": {
+            "filter" : {"activeUser" : {"$gt" : "data"}},
+            "sort" : {"userId" : 1}
+          }
+        }
+        """;
+      JsonNodeFactory nodefactory = objectMapper.getNodeFactory();
+      final ArrayNode arrayNode = nodefactory.arrayNode(testDatas.size());
```

Comment:
Not needed as no result docs should be received? similarly `ids`, `testDatas` not needed.